### PR TITLE
Release v1.1.1

### LINKED ISSUE #647: Decide whether next update release should be 1.1.1 or 1.2
There are a number of changes in the [draft specification](https://ocfl.io/draft/spec/) since the [1.1 release](https://ocfl.io/1.1/spec/) -- diff can be seen by change in [`draft/spec.index.md` from tag 1.1 to now](https://github.com/OCFL/spec/compare/1.1...main#diff-e7ec5397fdcc259e6a37f8918048c189b755143063257dd519f6402090c68867). This drift has led to confusion (e.g. https://github.com/OCFL/spec/issues/646)

Most changes are simple clarifications and formatting fixes which would suggest a 1.1.1 release. 

However, in response to https://github.com/OCFL/spec/issues/528 we removed a MUST that was agreed to be meaningless. Changed text around line 766 from:
> 3. File paths and filenames in the OCFL are case sensitive. Filesystems MUST preserve the case of OCFL filepaths and filenames.

to:
> 3. File paths and filenames in the OCFL are case sensitive. Implementations over filesystems that either do not preserve case or are not case sensitive require great care, including making appropriate choices for file paths and filenames.

Does this warrant 1.2?

### PR DESCRIPTION
**Pull request by zimeon**
Fixes #647

### CODE REVIEW

**Comment by zimeon** [comment]
Summary:
* `/1.1.0/` directory created with previous v1.1
* `/1.1.1/` directory created with redirects to current v.1.1
* `/1.1/` files updated from `drafts`, dates updates for spec and implementation notes, version history tables added at end, change log extended to cover 1.1.0->1.1.1 as well as 1.0->1.1.0
* `/news/` updates with new item and linked from `/`

**Review by neilsjefferies — CHANGES_REQUESTED** [review]
1.1/spec/change-log.md line 126: change->changed
1.1/spec/change-log.md line 138: updates->updated
1.1/spec/index.md line 1398: change log for 1.1.1 path shouldn't be 1.1

File: 1.1/spec/change-log.md [hunk 1]
```diff
@@ -106,3 +114,31 @@ The E048 error description in `validation-codes.md` is corrected to remove menti
...
+
+## Changes from OCFL v1.1.0 to v1.1.1
+
+[Version 1.1.1 of the OCFL Specification](https://ocfl.io/1.1/spec/) is a [patch version](https://semver.org/) update to the [OCFL Specification v1.1.0](https://ocfl.io/1.1.0/spec/). There are only clarifications.
+
+### Clarifications in v1.1.1
+
+#### Reword filesystem case sensitivity comments
+
+Version 1.1.0 had an unenforceable MUST regarding filesystem case preservation. The [Filesystem Features](https://ocfl.io/1.1/spec/#filesystem-features) section was change to instead point out that implementation over filesystems that either do not preserve case or are not case sensitive require great care, including making appropriate choices for file paths and filenames. (Issue [#528](https://github.com/OCFL/spec/issues/528))
```
Comment (rosy1280) [inline 1.1/spec/change-log.md:126]: change->changed

File: 1.1/spec/change-log.md [hunk 2]
```diff
@@ -106,3 +114,31 @@ The E048 error description in `validation-codes.md` is corrected to remove menti
...
+
+The range of specification sections (3.2 through 3.9) specifying the (Object Structure)[#object-structure] was added to make that explicit. (Issue [#637](https://github.com/OCFL/spec/issues/637))
+
+#### Clarify description of fixity in Implementation Notes
+
+The [fixity](https://ocfl.io/1.1/implementation-notes/#fixity) section of the Implementation Notes has been updated to point out differences in requirements for digests used for content addressing (the manifest and state blocks) and fixity (the fixity block). The section also notes that fixity algorithms may generate the same value for different file content. (Issue [#629](https://github.com/OCFL/spec/issues/629))
+
+#### Update links to Pairtree and NAMASTE specifications
+
+Links to both the Pairtree and NAMASTE specifications have been updates in the [Specification references](https://ocfl.io/1.1/spec/#references) and the [Implementation Notes references](https://ocfl.io/1.1/implementation-notes/#references). (Issues [#627](https://github.com/OCFL/spec/issues/627) and [#629](https://github.com/OCFL/spec/issues/629#issuecomment-1623865455))
```
Comment (rosy1280) [inline 1.1/spec/change-log.md:138]: updates->updated

File: 1.1/spec/index.md [hunk 3]
```diff
@@ -1373,3 +1389,10 @@ Registrations. S. Leonard. IETF. March 2016. URL:  <https://www.rfc-editor.org/r
 
 <span id="ref-rfc8141"/>**\[RFC8141]** Uniform Resource Names (URNs). P. Saint-Andre; J. Klensin. IETF. April 2017.
 Proposed Standard. URL: <https://www.rfc-editor.org/rfc/rfc8141>
+
+## Revision history
+
+| Version | Date | Description |
+| --- | --- | --- |
+| [v1.1.0](https://ocfl.io/1.1.0/spec/) | 7 October 2022 | First v1.1, [release notes](https://ocfl.io/news/#version-11-of-the-oxford-common-file-layout-ocfl-released), [change log](https://ocfl.io/1.1.0/spec/change-log.html) |
+| [v1.1.1](https://ocfl.io/1.1.1/spec/) | 8 November 2024 | Clarifications, [release notes](https://ocfl.io/news/#version-111-of-the-oxford-common-file-layout-ocfl-released), [change log](https://ocfl.io/1.1/spec/change-log.html) |
```
Comment (rosy1280) [inline 1.1/spec/index.md:1398]: (https://ocfl.io/1.1/spec/change-log.html)->(https://ocfl.io/1.1.1/spec/change-log.html)

**Review by rosy1280 — CHANGES_REQUESTED** [review]
being more clear than neil

File: 1.1/spec/change-log.md (hunk 1 above)
Comment (zimeon) [reply 1.1/spec/change-log.md]: Fixed in https://github.com/OCFL/spec/pull/654/commits/22f993007a0db2ecc67f1118d29fc5cb7aeb0a7c

File: 1.1/spec/index.md (hunk 3 above)
Comment (zimeon) [reply 1.1/spec/index.md]: Fixed in https://github.com/OCFL/spec/pull/654/commits/22f993007a0db2ecc67f1118d29fc5cb7aeb0a7c

**Review by neilsjefferies — DISMISSED** [review]

**Review by rosy1280 — DISMISSED** [review]

**Review by awoods — APPROVED** [review]

**Review by neilsjefferies — APPROVED** [review]

**Review by rosy1280 — APPROVED** [review]

**Pull request merged**